### ISSUE TITLE
.deb?

### ISSUE THREAD

**Issue by kalligator** [Dirli/pantheon-meteo#2]
Can you please provide a .deb for this?

**Comment by Dirli** [comment]
Yes, I thought about it. But I don't know how to implement it better yet. Unfortunately, i can't add one .deb for different versions of the distribution. I also don't use Ubuntu. I'll look for a solution to this problem later
